scripts/vagrant/zeppelin-dev/roles/maven/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+# setup maven
+# Assumes Ubuntu Server 14.04 LTS (HVM)
+# Assumes Java 7+ jdk installed
+---
+- name: download maven
+  get_url: url=http://apache.mirrorcatalogs.com/maven/maven-3/3.3.3/binaries/apache-maven-3.3.3-bin.tar.gz
```

Comment:
http://www.apache.org/dyn/closer.cgi returns a web page, and is not a direct URI to the archives.
If there is something that returns a true URI, let me know.  I can research updating.